tools/cmake/README.md

```diff
@@ -75,3 +75,24 @@ If you're running CMake directly, you may need to pass it in yourself as follows
 ```
 cmake -S <source-dir> -B <build-dir> -DCMAKE_BUILD_TYPE=debug
 ```
+
+## How to build a greentea test
+
+Install prerequisites suggested in the previous section and follow the below steps to build:
+* Generate the `.mbedbuild/` configuration directory for the Mbed target you want to run the test on using [mbed-os-example-blinky](https://github.com/ARMmbed/mbed-os-example-blinky)
+```
+$ mbedtools configure -t <TOOLCHAIN> -m <MBED_TARGET> 
+```
+* Copy `.mbedbuild/` into the test suite directory.
+* Set your current directory to the test suite directory
+* Run the following command to build the test binary with the full profile
+
+  ```
+  touch mbed-os.lib && mkdir build && cd build && cmake .. -G Ninja && cmake --build .
```

Comment:
```suggestion
  touch mbed-os.lib && mkdir cmake_build && cd cmake_build && cmake .. -G Ninja && cmake --build .
```
To be in line with what `mbedtools` creates.

Reply:
@rajkan01 Can you make the requested edits ?